sites-available/README.md

```diff
@@ -9,4 +9,9 @@ example.com (handles traffic from both www.example.com and example.com)
 foobar.com (as above)
 test.foobar.com (handles traffic from both www.test.foobar.com and test.foobar.com)
 ```
+The HTTPS no-default server config requires a valid certificate (self signed will suffice).
+One can generate the certificate with: 
+```
+sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/nginx/ssl/nginx.key -out /etc/nginx/ssl/nginx.crt
```

Comment:
This doesn't really work anymore since Common Name (CN) is deprecated in favor of Subject Alternative Name (SAN). [http://stackoverflow.com/a/5937270](http://stackoverflow.com/a/5937270)